turbinia/workers/__init__.py

```diff
@@ -318,9 +318,11 @@ def deserialize(cls, input_dict):
     Returns:
       TurbiniaTask: Deserialized object.
     """
+    from turbinia import client  # Avoid circular imports
+
     type_ = input_dict['name']
     try:
-      task = getattr(sys.modules[__name__], type_)()
+      task = getattr(sys.modules['turbinia.client'], type_)()
```

Comment:
Cool, yeah, that's nicer than having to re-import everything (assuming we don't have any cycles like codefactor thinks).